phpcs.xml

```diff
@@ -2,26 +2,31 @@
 <ruleset name="WirecardOxidRuleset">
   <description>Wirecard OXID extension</description>
 
-  <!-- Ignore patterns -->
-  <exclude-pattern>*/Application/views/*</exclude-pattern>
-  <exclude-pattern>*/oxfunctions.php*</exclude-pattern>
-  <exclude-pattern>*/Core/Smarty/*</exclude-pattern>
-  <exclude-pattern>*/Core/tcpdf/*</exclude-pattern>
-  <exclude-pattern>*/Core/adodblite/*</exclude-pattern>
-  <exclude-pattern>*/out/*</exclude-pattern>
-  <exclude-pattern>*/Setup/tpl/*</exclude-pattern>
-  <exclude-pattern>*/tmp[^/]*/*</exclude-pattern>
-  <exclude-pattern>/config.inc.php</exclude-pattern>
-  <exclude-pattern>/vendor/</exclude-pattern>
-  <exclude-pattern>/Tests/</exclude-pattern>
-  <exclude-pattern>/tests/</exclude-pattern>
-  <exclude-pattern>/translations/</exclude-pattern>
-  <exclude-pattern>/views/</exclude-pattern>
-  <exclude-pattern>/phpmd/</exclude-pattern>
-  <exclude-pattern>.phpstorm.meta.php/*</exclude-pattern>
-  <exclude-pattern>*/de/</exclude-pattern>
-  <exclude-pattern>*/en/</exclude-pattern>
-  
+    <!-- Ignore patterns -->
```

Comment:
intent size should be 2 like defined in the .editorconfig

Reply:
Fixed :)